### ISSUE TITLE
Excellent! – thanks a lot

### ISSUE THREAD

**Issue by lrosenth** [LaDHUL/Knora-projects-metadata#1]
I looked at the concept and it seems to  me to fit perfectly to our needs. There is just very few question I would like to open for discussion:

- we do not have datasets in the classical way. Do I get it right that what we call a "project" or a "database" would be the equivalent for Knora?
- from Your experience, do we need any additional features because we do not have "classical" datasets?

As soon as I'm back from vacation I will start implementing it together with Ivan, André and Ben (backend and GUI). I'll make an issue in the knora and GUI githubs. As far as I could check the Ontology is very well done (may be some minor issues arrive when we start implementing/using it – but so far I cannot see anything...) Who would like to join us in implementing it? @loicjaouen ? @mrivoal ?

**Comment by mrivoal** [comment]
> we do not have datasets in the classical way. Do I get it right that what we call a "project" or a "database" would be the equivalent for Knora?

Yes, to me a `knora-base:knoraProject` would be a `dcat:Dataset` 

 `dcat:Dataset rdfs:comment "A collection of data, published or curated by a single source, and available for access or download in one or more formats"@en;`

and both the data named graph and Sipi files could be 2 `dcat:Distribution` amongst others.
 ` dcat:Distribution rdfs:comment """Represents a specific available form of a dataset. Each dataset might be available in different forms, these forms might represent different formats of the dataset or different 
 endpoints. Examples of distributions include a downloadable CSV file, an API or an RSS feed"""@en;`

And sure, @loicjaouen and I would be happy to help if we can!
